src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        Random random = new Random();
+        ColorSupplier supplier = new ColorSupplier();
+        int number = random.nextInt(100);
+        String color = String.valueOf(supplier.getRandomColor());
```

Comment:
Directly using `supplier.getRandomColor().name()` is preferred over `String.valueOf(supplier.getRandomColor())` to get the String representation of the enum.